### ISSUE TITLE
Lock files out of date

### ISSUE THREAD

**Issue by dimaqq** [python/black#860]
```
… ~/black> git rev-parse HEAD
a4399f3054206eac52a89832a415e31077bafa0a

… ~/black> pipenv install
Pipfile.lock (1d69ac) out of date, updating to (1480f9)…
[...]
```

The lock file diff is quite large and I'm not sure what was the relevant change.
Last commit to touch Pipefile was 866be066463fc8fd01c16559596641f6ead1e797
@fried @willingc

**Comment by zsol** [comment]
Fixed in 5b01a8e3b07f2bbf4b0bd9071853364254f308ca

**Comment by zsol** [comment]
as a sidenote, `Pipfile.lock` might be different on each platform, and the commit above was generated using python 3.7.3 on a 64bit Linux platform
